doctr/io/elements.py

```diff
@@ -298,6 +374,18 @@ def synthesize(self, **kwargs) -> List[np.ndarray]:
 
         return [page.synthesize() for page in self.pages]
 
+    def export_as_xml(self, file_title: str = 'docTR - XML export (hOCR)', **kwargs) \
```

Comment:
no need for `file_title`, the kwargs will let the user specify the file_title if they want